rasa/core/processor.py

```diff
@@ -188,19 +205,49 @@ async def get_tracker_with_session_start(
             sender_id: Conversation ID for which to fetch the tracker.
 
         Returns:
-              Tracker for `sender_id` if available, `None` otherwise.
+              Tracker for `sender_id`.
         """
+        # TODO: Remove in Rasa Open Source 3.0
+        rasa.shared.utils.io.raise_warning(
```

Comment:
I'd personally not consider the `processor` module a public API (in contrast to `Agent`). So imo fine to just drop it. What do you think?

Reply:
👍